content/ember/v1/ember-reduce-computed-property.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: ember-reducecomputed-ember-arraycomputed
+id: ember-reducecomputedproperty-ember-arraycomputedproperty
```

Comment:
I'm not sure we can actually change the ids on anything because they are linked to by ember-cli console.log warnings. 

I'm not sure but it should be checked before we do update any ids.

Reply:
Thanks for the explanation! It seemed like this `id` was wrong based on the old app? But I wasn't sure if the current emberjs.com website url and `id` were necessarily linked?

re: https://www.emberjs.com/deprecations/v1.x/#toc_ember-reducecomputed-ember-arraycomputed and 
https://www.emberjs.com/deprecations/v1.x/#toc_ember-reducecomputedproperty-ember-arraycomputedproperty

Reply:
so there are some deployment issues that I'm trying to sort out on the new deprecations app right now but essentially you should be able to see what the links look like you can prefix the # link with `https://deprecations-app-prod.herokuapp.com/ember/v1.x/` (it should work the first time and then break on refresh but that's expected right now 😖) 

so you will notice that if you go to https://deprecations-app-prod.herokuapp.com/ember/v1.x/#toc_ember-reducecomputed-ember-arraycomputed it doesn't scroll down the page but going to https://deprecations-app-prod.herokuapp.com/ember/v1.x/#toc_ember-reducecomputedproperty-ember-arraycomputedproperty does ... or at least it should... blast it! 😖 

So I'm not exactly sure exactly what the **right** way is... but the thing I *know* is that we need to maintain inbound links. It's potentially a good idea to raise the issue of QA right now to make sure we are in fact supporting the same links 🤔